README.md

```diff
@@ -1,6 +1,8 @@
 midnight/package-cohesion
 =========================
-This is a script that checks the cohesion of your composer package.
+This script checks how cohesive your Composer package is. It shows you file dependency clusters. If you've got a single cluster, your all good. Your package is very cohesive. If you've got one big cluster and several small ones, you should be fine. If you've got two or more larger clusters, you probably want to split your package into several smaller packages.
```

Comment:
Can I suggest some grammar and punctuation changes?

```suggestion
This script checks how cohesive your Composer package is, i.e., your file dependency clusters.

If you've got a single cluster, you're all good, your package is very cohesive.
If you've got one big cluster and several small ones, you should be fine.
If you've got 2 or more larger clusters, you probably want to split your package into several smaller packages.
```

Reply:
Great changes! Could you open a separate PR please?

Reply:
@MidnightDesign, there you go: https://github.com/MidnightDesign/package-cohesion/pull/2.